opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -182,7 +182,7 @@ def _get_time_from_ns(nanoseconds: int) -> Dict:
     nanoseconds"""
     if not nanoseconds:
         return None
-    seconds, nanos = divmod(nanoseconds, 1e9)
+    seconds, nanos = divmod(nanoseconds, 10 ** 9)
```

Comment:
Can you create a constant like `NANOS_IN_SECONDS` that we have in the metrics exporter. We need to find a way to reuse code between the two exporters :/

Reply:
I'm assuming this gets converted into a proto `Timestamp` object eventually. Alternatively, you might be able to use `Timestamp.FromNanoseconds()` in place of this whole functions, e.g.
https://github.com/open-telemetry/opentelemetry-python/blob/5ff9600a62f9c82fc120b03e5729a53341465626/ext/opentelemetry-ext-opencensusexporter/src/opentelemetry/ext/opencensusexporter/util.py#L46